conf/development.ini

```diff
@@ -46,11 +48,26 @@ level = DEBUG
 handlers =
 qualname = lms
 
+[logger_exc_logger]
+level = ERROR
+handlers = exc_handler
+qualname = exc_logger
+propagate = 0
```

Comment:
All these `ini` file changes are straight from the [`pyramid_exclog` docs](https://docs.pylonsproject.org/projects/pyramid_exclog/en/latest/). The one addition is `propagate = 0` which causes `pyramid_exclog` to swallow the exception instead of letting it propagate. Otherwise it propagates up to `gunicorn` which _also_ prints the exception and stacktrace so everything gets printed twice.